### PR TITLE
[MIG] Migrated purchase commercial partner to v13.

### PR DESCRIPTION
**Pull request by sudhir-erpharbor**

### CODE REVIEW

File: purchase_commercial_partner/views/purchase.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (newtratip) [inline purchase_commercial_partner/views/purchase.xml]: Please change file's name to purchase_views.xml
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#file-naming

**Review by newtratip — CHANGES_REQUESTED** [review]
@sudhir-erpharbor Small change.

**Comment by newtratip** [comment]
@sudhir-erpharbor Could you squash your commit ?

**Comment by sudhir-erpharbor** [comment]
> @sudhir-erpharbor Could you squash your commit ?

Done. Thanks for the review.

**Review by newtratip — APPROVED** [review]
LGTM :+1:

**Review by CasVissers-360ERP — APPROVED** [review]
LGTM, functional test.

**Pull request merged**